### PR TITLE
[Feature] Dynamic parameters on UI - first version

### PR DESCRIPTION
**Pull request by mproch**
First version of handling dynamic parameters on FE. It works, but there are still some UX quirks and refactorings to be done:
- default values are not filled for dynamic parameters
- to display dynamic parameters correctly request to BE is done when modal is opened
- after adding parameters additional request to BE is done to validate them
- we filter out MissingParameters validation errors for parameters that are going to be added after handling request 

I'd like to leave those for next PRs, as this version is functional and I don't want to make it too large

dynamic2

### CODE REVIEW

File: ui/client/components/graph/node-modal/NodeDetailsContent.js [hunk 1]
```diff
@@ -524,9 +535,15 @@ export class NodeDetailsContent extends React.Component {
...
   }
 
-  createExpressionField = (fieldName, fieldLabel, expressionProperty, fieldErrors) => this.doCreateExpressionField(fieldName, fieldLabel, `${expressionProperty}`, fieldErrors)
+  //this is for "static" fields like expressions in filters, switches etc.
+  createStaticExpressionField = (fieldName, fieldLabel, expressionProperty, fieldErrors) => {
+    return this.doCreateExpressionField(fieldName, fieldLabel, `${expressionProperty}`, fieldErrors)
+  }
 
-  createExpressionListField = (fieldName, expressionProperty, listFieldPath, fieldErrors, fieldType) => this.doCreateExpressionField(fieldName, fieldName, `${listFieldPath}.${expressionProperty}`, fieldErrors, fieldType)
+  //
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/NodeDetailsContent.js:543]: //? :)

File: ui/client/components/graph/node-modal/NodeDetailsContent.js [hunk 2]
```diff
@@ -220,7 +231,7 @@ export class NodeDetailsContent extends React.Component {
               editedNode={this.state.editedNode}
               savedNode={this.state.editedNode}
               setNodeState={newParams => this.setNodeDataAt("ref.parameters", newParams)}
-              createListField={(param, index) => this. createExpressionListField(
+              createListField={(param, index) => this. createParameterExpressionField(
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/NodeDetailsContent.js:234]: there is not necessary space at `this. `

File: ui/client/components/graph/node-modal/NodeDetailsContent.js (hunk 1 above)
Comment (mproch) [reply ui/client/components/graph/node-modal/NodeDetailsContent.js]: I added what I forgot to add :)

File: ui/client/components/graph/node-modal/NodeDetailsContent.js (hunk 2 above)
Comment (mproch) [reply ui/client/components/graph/node-modal/NodeDetailsContent.js]: :+1:

**Pull request merged**